apis/core/lib/domain/csv-source/CsvSource.ts

```diff
@@ -38,15 +46,23 @@ export default function Mixin<Base extends Constructor<Omit<CsvSource, keyof Api
       }) as any
     }
 
-    setDialect(dialect: Partial<Csvw.Dialect>): void {
-      this.pointer.out(csvw.dialect).deleteOut()
```

Comment:
This shouldn't have been removed 🤔

Reply:
Ah, no we're good because the rewrite below will update changed properties but leave any surplus. Maybe it's a bad thing considering `PUT` but 🤷